src/installation/guides/fde.md

```diff
@@ -137,8 +137,10 @@ Do you want to import this public key? [Y/n] y
 ...
 ```
 
-UEFI systems will have a slightly different package selection. The installation
-command for a UEFI system will be as follows.
+UEFI systems will have a slightly different package selection.
+On some devices the EFI firmware is 32 bits: install grub-i386-efi instead
+To confirm EFI firmware architecture: cat /sys/firmware/efi/fw_platform_size
+The installation command for a UEFI system will be as follows.
```

Comment:
If possible, this informatio should be transferred to the chroot guide as well. Please use inline code for commands and package names, too.

```
install `grub-i386-efi` instead
```